src/Enum/Region.php

```diff
@@ -4,60 +4,59 @@
 
 namespace ToshY\BunnyNet\Enum;
 
-enum Region
+enum Region: string
 {
-    /** Falkenstein / Frankfurt (Germany) | Main */
-    case DE;
-    case FS;
+    /** Falkenstein / Frankfurt (Germany) | Main  */
+    case DE = 'DE'; // placeholder value
```

Comment:
Comment is I think unneccesary; the issue already has a comment that denotes this